config.toml

```
@@ -8,6 +8,11 @@ logo = "/img/breadtube.png"
 targetBlank = true
 defaultTags = []
 
+[languages]
+  [languages.en]
+    title = "English"
```

This is changing the site title.
```suggestion
    title = "BreadTube"
```